recipe/meta.yaml

```diff
@@ -10,11 +10,8 @@ source:
 
 build:
   number: 0
```

Comment:
I fixed the missing build number bump while merging: https://github.com/conda-forge/pythran-feedstock/commit/5207e12049bf81a1708ddf1db908e890f5a23e4b